### PR TITLE
Chaplain Vending Machine

### PR DESCRIPTION
**Pull request by SgtManGuy**
[CLOTHING] [MAPPING] [REWORK]

## About the PR 
Second pr, pulled the first one down because I messed some stuff up that I wasn't able to fathom. New PR should be less messed up and have code that's more inline with current standards.
This PR adds a new chaplain vending machine the replaces the current locker with all of the old locker's contents.
It also has several new outfits and books that can be opened. Presently, the books don't have any function outside of looking pretty.

## Why's this needed? 
Chaplain is a role that's lacking in a few areas and relies heavily upon the player creating their own fun. Hopefully, these new items will make that east for chaplains to do so.
Skeleton book sprites by Rubi. X, red/white, red/dark, blue/white book sprites by Paco. Descriptions by Large Amounts of Screaming and GibusGame, assistance from Tyrant.

## Changelog 

```changelog
(u)What/SgtManGuy, Tyrant, Large Amounts of Screaming, Rubi, Paco, GibusGame
(*)Replaces the chaplain's locker with a vending machine.
(+)The new vending machine, the Deus Ex Machina contains all of the old locker's contents.
(+)There are 5 new outfits and many new prop books in the vending machine.

### CODE REVIEW

**Comment by SgtManGuy** [comment]
Screenshot 2024-02-13 084457
Screenshot 2024-02-06 192434

**Comment by GrunkaBunka** [comment]
I really can't comment on the code, but from a purely game design position, I love this idea.

File: code/obj/item/scripture.dm [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+			if (!src.open)
+				src.open(user)
+			else
+				src.close(user)
+			user.update_inhands()
+
+	proc/open (mob/user as mob)
+		src.open = TRUE
+		icon_state = "[icon_state]open"
+		item_state = "[item_state]open"
```
Comment (Valtsu0) [inline code/obj/item/scripture.dm:24]: I don't like a var having the same name as a proc.

Maybe call it is_open

Comment (SgtManGuy) [reply code/obj/item/scripture.dm]: Should be changed, with thanks to gallowscalibrator

File: code/obj/item/scripture.dm [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+/obj/item/scripture/bluehb
+	name = "last words of the outlander"
+	icon_state = "bluehb"
+	item_state = "bluehb"
+	desc = "A text in a spacefaring language lost to the galaxy. It contains diagrams of its history that are difficult to parse."
+	item_state_base = "bluehb"
+
+/obj/item/scripture/skeletonhb
+	name = "bone tone"
```
Comment (Valtsu0) [inline code/obj/item/scripture.dm:81]: Did you mean bone tome?

Comment (Valtsu0) [inline code/obj/item/scripture.dm:81]:
```suggestion
	name = "bone tome"
```

Everything else looks good

**Review by Valtsu0 — COMMENTED** [review]
image
This doesn't look intentional

File: code/obj/item/scripture.dm (hunk 2 above)
Comment (SgtManGuy) [reply code/obj/item/scripture.dm]: Solved, thanks for catching this.

**Comment by SgtManGuy** [comment]
> image This doesn't look intentional

It indeed was not. I may have fucked up while resolving a conflict, should be fixed now.

**Pull request merged**